### PR TITLE
Update UserGuide on list and FAQ

### LINKED ISSUE #70: Update User Guide

### PR DESCRIPTION
**Pull request by Vinci-Hu**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,11 +29,50 @@ Example of usage:
...
+* There should not be any descriptions after keyword `list`.  
+
+Example of usage: 
+
+`list`
+
+### Display the list of foods by category: `list <CATEGORY>`
+Displays the list of food items under specified category stored in the fridge.
+
+Format: `todo CATEGORY_NAME`
```
Comment (SimJJ96) [inline docs/UserGuide.md]: I think you have missed out on replacing the word todo with list

Comment (kwokyto) [inline docs/UserGuide.md]: not `todo`

**Review by kwokyto — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (Vinci-Hu) [reply docs/UserGuide.md]: Fixed

Comment (Vinci-Hu) [reply docs/UserGuide.md]: Yes thank you.

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Review by Leeyp — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -9,13 +9,11 @@
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+1. Download the latest version of `Duke` from [here](http://link.to/duke).
```
Comment (joohwan58) [inline docs/UserGuide.md]: Duke? not FridgeFriend?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -9,13 +9,11 @@
...
 
 ## Features 
 
-{Give detailed description of each feature}
-
-### Adding a todo: `todo`
+### (Example)Adding a todo: `todo`
 Adds a new item to the list of todo items.
```
Comment (joohwan58) [inline docs/UserGuide.md]: outdated? we dont have todos

File: docs/UserGuide.md [hunk 4]
```diff
@@ -29,11 +27,50 @@ Example of usage:
...
+
+`list idontknow`
+
 ## FAQ
 
 **Q**: How do I transfer my data to another computer? 
 
-**A**: {your answer here}
+**A**: Copy the `.jar` file along with `data` folder to the target computer.
+As long as the target computer satisfies our project prerequisites, it can run with the saved data as before.
```
Comment (joohwan58) [inline docs/UserGuide.md:69]: the 'save' folder

**Review by joohwan58 — CHANGES_REQUESTED** [review]

**Review by joohwan58 — APPROVED** [review]
LGTM

**Pull request merged**